app/js/trouble.js

```diff
@@ -72,52 +88,58 @@
 
 		function open() {
 			$dropdown.removeClass('no_transition');
-			$dropdown.data('status', 'open")'
+			$dropdown.data('status', 'open');
 			$dropdown.addClass('gbl_dropdown_active');
-			$dropdown.focus()
+			$dropdown.focus();
 			$toggle.addClass('gbl_dropdown_active');
 			$dropdown.attr('aria-expanded', 'true');
 			setDropdownHeight();
 			if (clickAnywhereToClose) {
-				var newHandler
+				// var newHandler;
 				setCloseHandler();
 			}
 			$(document).trigger(ns + 'Open');
 		}
 
+		function getStatus() {
+			return {
+				status: "open"
+			};
+		}
+
+		function setStatus() {
+			newStatus = "closed";
+		}
+
 		function toggleDropdown(e) {
-			e.preventDefault()
-			e.stopPropagation()
-			if ($dropdown.data('status') =='closed') {
-				function setStatus() {
-					newStatus = "closed"
-				}
+			e.preventDefault();
+			e.stopPropagation();
+
+			if ($dropdown.data('status') === 'closed') {
+				setStatus();

```

Comment:
In this case, setStatus doesn't need to be called as it wasn't called in the original code. The removal of the functions declared in the conditional blocks was spot on.